src/main/java/manga/Manga.java

```diff
@@ -0,0 +1,44 @@
+package manga;
+
+import author.Author;
+
+import java.util.Objects;
+
+public class Manga {
+    private String mangaName;
+    private Author author;
+
+    public Manga(String mangaName, Author author) {
+        this.mangaName = mangaName;
+        this.author = author;
+    }
+
+    public String getMangaName() {
+        return mangaName;
+    }
+
+    public Author getAuthor() {
+        return author;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) {
+            return true;
+        }
+
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
```

Comment:
Similar comment for Author.java. Not sure if this check is intentional or requires another set of parentheses.